easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'Qhull'
+version = '2020.2'
+
+homepage = 'http://www.qhull.org'
+
+description = """
+ Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,
+ halfspace intersection about a point, furthest-site Delaunay triangulation,
+ and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
+ higher dimensions. Qhull implements the Quickhull algorithm for computing the
+ convex hull.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.qhull.org/download/']
+sources = ['%(namelower)s-%(version_major)s-src-8.0.2.tgz']
```

Comment:
@SimonPinches please add the missing checksum here.

Reply:
@verdurin Done :-)